src/models/generated/FACTSControlDevice.jl

```diff
@@ -0,0 +1,115 @@
+#=
+This file is auto-generated. Do not edit.
+=#
+
+#! format: off
+
+"""
+    mutable struct FACTSControlDevice <: ElectricLoad
+        name::String
+        available::Bool
+        bus::ACBus
+        mode::Int64
```

Comment:
Call this control mode and lets make this an enum so we don't need to guess from the integer